docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: '3.9'
+services:
+  nginx:
+    build: ./nginx
+    ports:
+      - 8008:80
+    depends_on:
+      - web
+  web:
+    build: 
+      context: .
+    command: gunicorn game_nerd.wsgi:application --bind 0.0.0.0:8000
```

Comment:
you'll also need to run a migrations to start the app?

`python3 manage.py migrate && gunicorn game_nerd.wsgi:application --bind 0.0.0.0:8000`

Additionally after `migrate` you'll need to load games from your fixtures.

There's simple way to verify if your setup is working. 
Remove project directory locally, clone it again into new folder and run `docker-compose up`. If you're able to view your app in browser, then docker setup is correct. Could you please do this?